src/test/java/uk/nhs/prm/deduction/e2e/utility/TestUtils.java

```diff
@@ -57,9 +66,31 @@ public static boolean excludeComparisons(Node node) {
             // return false to skip comparison when the type of compared value is in the excludedList
             return !(node.getNodeName().equals("id") && excludeList.contains(idRootValue));
         }
+
+        // return false to skip comparison in case when the node name is "message-id"
+        if (node.getNodeName().equals("message-id")) {
+            return false;
+        }
+
         return true;
     }
 
+    public static void addRecordToTrackerDb(TrackerDb trackerDb, String inboundConversationId, String largeEhrCoreMessageId, String nhsNumberForTestPatient, String previousGpForTestPatient, String state) {
+        String timeNow = ZonedDateTime.now(ZoneOffset.ofHours(0)).toString();
+
+        trackerDb.save(new TransferTrackerDbMessage(
```

Comment:
Just a thought, would it be beneficial implementing the builder pattern for TransferTrackerDbMessage, given the amount of parameters it takes via the constructor? Could be out of scope but I thought I'd chuck it in! :D